src/SprykerSdk/Spryk/Model/Spryk/Builder/Transfer/TransferSpryk.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Builder\Transfer;
+
+use DOMDocument;
+use SimpleXMLElement;
+use SprykerSdk\Spryk\Exception\XmlException;
+use SprykerSdk\Spryk\Model\Spryk\Builder\SprykBuilderInterface;
+use SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface;
+use SprykerSdk\Spryk\Style\SprykStyleInterface;
+use Zend\Filter\FilterChain;
+use Zend\Filter\Word\DashToCamelCase;
+
+class TransferSpryk implements SprykBuilderInterface
+{
+    public const ARGUMENT_TARGET_PATH = 'targetPath';
+    public const ARGUMENT_NAME = 'name';
+    protected const SPRYK_NAME = 'transfer';
+
+    /**
+     * @var string
+     */
+    protected $rootDirectory;
+
+    /**
+     * @param string $rootDirectory
+     */
+    public function __construct(string $rootDirectory)
+    {
+        $this->rootDirectory = $rootDirectory;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return static::SPRYK_NAME;
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface $sprykDefinition
+     *
+     * @return bool
+     */
+    public function shouldBuild(SprykDefinitionInterface $sprykDefinition): bool
+    {
+        return !$this->isTransferDefined($sprykDefinition);
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface $sprykDefinition
+     * @param \SprykerSdk\Spryk\Style\SprykStyleInterface $style
+     *
+     * @return void
+     */
+    public function build(SprykDefinitionInterface $sprykDefinition, SprykStyleInterface $style): void
+    {
+        $xml = $this->getXml($sprykDefinition);
+
+        $dashToCamelCaseFilter = $this->getDashToCamelCase();
+
+        $transferName = $dashToCamelCaseFilter->filter($this->getArgumentName($sprykDefinition));
+
+        $transfers = $xml[0];
+        if (!$transfers) {
+            return;
+        }
+
+        $transfers->addChild('transfer')
```

Comment:
is there a way to force <transfer></transfer> and not <transfer /> ?

Reply:
wym?

Reply:
wtf the comment got messed up.. i meant force <transfer></transfer> and not <transfer/>

Reply:
WHY IS IT NOT GETTING IN!! I mean `<transfer></transfer>` and not `<transfer />`

Reply:
I will check